mdtk/degradations.py

```diff
@@ -20,6 +20,9 @@
 
 MAX_GAP_DEFAULT = 50
 
+MIN_VELOCITY_DEFAULT = 100
```

Comment:
Just checking why we want the default to make notes of exactly 100 velocity?

Reply:
Slightly silly reason: This is the only way to get ACME recreated exactly. Forcing the sampling of a velocity value changes the seed for the next excerpt. Not sure if this is important or not, but I'm just realizing if we leave it like this, we should probably add a note to the docs.

Reply:
Great, yup please just add a comment about it but I'm happy for this behaviour to be left as is.